frontend/packages/ux-editor/src/components/Properties/Properties.module.css

```
@@ -4,6 +4,10 @@
 }
 
 .datamodelBindings {
-  --vertical-padding: calc(var(--fds-spacing-5) - var(--studio-property-button-vertical-spacing));
-  padding: var(--vertical-padding) 0;
+  padding: var(--fds-spacing-5) 0;
+}
+
+.texts {
+  background-color: white;
```

What about using the fds token for the defualt surface? 😊 
```suggestion
  background-color: var(--fds-surface-neutral-default)
```